src/commands/deploy.rs

```diff
@@ -203,3 +204,24 @@ impl DeployCommand {
         Ok(bindle_id.clone())
     }
 }
+
+fn print_available_routes(
+    address: &str,
+    base: &str,
+    cfg: &spin_loader::local::config::RawAppManifest,
+) {
+    if cfg.components.is_empty() {
+        return;
+    }
+
+    println!("Available Routes:");
+    for component in &cfg.components {
+        if let TriggerConfig::Http(http_cfg) = &component.trigger {
+            let route = RoutePattern::from(base, &http_cfg.route);
+            println!("  {}: http://{}{}", component.id, address, route);
+            if let Some(description) = &component.description {
```

Comment:
I think we removing the description might make the output cleaner.

Reply:
I did this for consistency with the local printing of routes but can remove it if folks prefer!

Reply:
Yeah if we don't like the format it seems reasonable to change both (in a separate PR).

Reply:
sounds good